src/extension.ts

```diff
@@ -57,5 +78,8 @@ const calculateLengthOfImage = (rows: string[]) => {
   return Math.min(longestRow.length * 7, maxImageWidth);
 };
 
+
+
```

Comment:
Pull the latest `master` changes and run `yarn && yarn prettier --write src/extension.ts`